### PR TITLE
Create rust.yml

### PR DESCRIPTION
**Pull request by dburgener**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup
+      run: |
+              sudo apt-get update -q
+              sudo apt-get -y install secilc
+    - name: Build
+      run: cargo build --verbose
```
Comment (l0kod) [inline .github/workflows/rust.yml:24]: We should reuse [build.sh](https://github.com/dburgener/cascade/blob/main/.pipelines/build.sh) and similar scripts if possible, or at least their content (e.g. `rustup run stable cargo build --verbose`).

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - name: Run tests
+      run: cargo test --verbose
+    - name: Check format
+      run: cargo fmt --all -- --check
```
Comment (l0kod) [inline .github/workflows/rust.yml:28]: Other steps could be inspired from https://github.com/dburgener/cascade/blob/main/.pipelines/OneBranch.PullRequest.yml#L48-L85

**Review by l0kod — CHANGES_REQUESTED** [review]

**Comment by l0kod** [comment]
As suggested by @liuw (see (comment)[https://github.com/dburgener/cascade/pull/2#issuecomment-961509579]) we should also use `cargo clippy`. It would require to exclude the generated `parser.rs` from the analysis though.

**Comment by dburgener** [comment]
> As suggested by @liuw (see comment) we should also use cargo clippy. It would require to exclude the generated parser.rs from the analysis though, see lalrpop/lalrpop#632 (which is not yet published in crates.io).

I definitely agree with this. We should do it in a follow-up PR though so that we can get at least some CI working ASAP.

**Review by l0kod — APPROVED** [review]

**Pull request merged**